spec.html

```diff
@@ -129,33 +129,22 @@ <h1>
     <emu-clause id="sec-getiteratorflattenable" type="abstract operation">
       <h1>
         GetIteratorFlattenable (
-          _obj_: an ECMAScript language value,
-          _hint_: ~sync~ or ~async~,
+          _obj_: an ECMAScript language value
         ): either a normal completion containing an Iterator Record or a throw completion
       </h1>
       <dl class="header">
       </dl>
       <emu-alg>
         1. If _obj_ is not an Object, throw a *TypeError* exception.
-        1. Let _alreadyAsync_ be *false*.
-        1. Let _method_ be *undefined*.
-        1. If _hint_ is ~async~, then
-          1. Set _method_ to ? Get(_obj_, @@asyncIterator).
-          1. Set _alreadyAsync_ to *true*.
-        1. If IsCallable(_method_) is *false*, then
-          1. Set _method_ to ? Get(_obj_, @@iterator).
-          1. Set _alreadyAsync_ to *false*.
-        1. If IsCallable(_method_) is *false*, then
+        1. Let _method_ be ? GetMethod(_obj_, @@iterator).
+        1. If _method_ is *undefined*, then
```

Comment:
This change from Get/IsCallable to GetMethod/undefined is technically normative, but it more closely matches the existing `GetIterator` behaviour. If `obj` has a `Symbol.iterator` property that is non-callable, it will now throw instead of falling back to treating `obj` as an iterator.

Reply:
I've removed this to split it into a separate PR.